mobile-app/app/screens/AppNavigator/screens/Balances/hooks/TokenLockedBalance.ts

```diff
@@ -15,10 +15,10 @@ export interface LockedBalance {
 
 /**
  *
- * @param symbol optional token symbol
- * @returns Map of all token's locked balance or single object of symbol passed
+ * @param symbol optional token displaySymbol
+ * @returns Map of all token's locked balance or single object of displaySymbol passed
```

Comment:
I think there's a reason why it was kept to Symbol. Might be the API?

Reply:
Can you double check first? Just to ensure we don't cause any regressions

Reply:
we kept `symbol` last time because `getTokenPrice()` requires symbol.
If you look at line 41, I am still calling `getTokenPrice()` using `symbol`, that part is unchanged, i only changed the `displaySymbol` to be used as the key for the returned `Map`

I have also cross-checked in mainnet, the number checks out for all of usd, DFI and BTC value of tokens, LPs, and dTokens